jekyll/_cci2_ja/server/operator/manage-virtual-machines-with-vm-service.adoc

```diff
@@ -72,7 +73,7 @@ vm_service:
 [.table.table-striped]
 [cols=2*, options="header", stripes=even]
 |===
-|リージョン
```

Comment:
@yuki-sakamoto-san Could you check which is the preferred translation here? they are shown as the same work "region" in google translate